test/api/users.js

```diff
@@ -0,0 +1,71 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const bcrypt = require('bcryptjs');
+const { expect } = chai;
+chai.use(chaiHttp);
+const { User } = require('../../models');
+const sinon = require('sinon');
+const authMiddleware = require('../../middlewares/checkAuth');
+let app;
+var Token = '';
+
+describe('## USERS', function () {
+    let authToken;
```

Comment:
```suggestion
    let authToken = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImpvaG5vZUBleGFtcGxlLmNvbSIsInVzZXJJZCI6IjVmMDQ0NTFlODBmMGJhMzk5ZGYxZjI2ZCIsInJvbGUiOiJ1c2VyIiwiaWF0IjoxNTk0MTE1Nzg4LCJleHAiOjE1OTQyMDIxODh9.VoSPs24TntksEk3fP-vV5VLYxmTOMHUa10SmGUt52DU;
 ```

initialize this variable

Reply:
ok